[fix][doc] Add link to PIP to Contribute page (#468)
apache/pulsar-site: contribute/about.md

If there’s an existing GitHub issue for your intended contribution, please comment about your intended work. Once the work is understood, a committer will assign the issue to you. If an issue is currently assigned, please check with the current assignee before reassigning.

For moderate or large contributions, you should not start coding or writing a design document, a.k.a PIP (see below) unless there is a corresponding GitHub issue assigned to you for that work. Simple changes, like fixing typos, do not require an associated issue.

### Pulsar Improvement Proposal (PIP)

> On any change which is either significant, or changes any public facing component of Pulsar (API, configuration, ...), the author should write design document known as Pulsar Improvement Proposal (PIP). There is a process for it that you should follow before starting to write one. It is [here]([url](https://github.com/apache/pulsar/blob/master/wiki/proposals/PIP.md))

Reviewer: ```suggestion
On any change which is either significant (e.g., new features), or changes any public-facing component of Pulsar (API, configuration, metric ...), you need to write a design document known as a *Pulsar Improvement Proposal (PIP)*. 

Before starting to write a PIP, make sure you've read [Pulsar PIP Guideline](https://github.com/apache/pulsar/blob/master/wiki/proposals/PIP.md). Then you can submit one with the [Pulsar PIP template](https://github.com/apache/pulsar/issues/new?assignees=&labels=PIP&template=pip.yml&title=PIP-XYZ%3A+).
```

> On any change which is either significant (e.g., new features), or changes any public-facing component of Pulsar (API, configuration, metric ...), you need to write a design document known as a *Pulsar Improvement Proposal (PIP)*. There is a process for it that you should follow before starting to write one. It is [here](https://github.com/apache/pulsar/blob/master/wiki/proposals/PIP.md)

Reviewer: ```suggestion
On any change which is either significant (e.g., new features), or changes any public-facing component of Pulsar (API, configuration, metric ...), you need to write a design document known as a *Pulsar Improvement Proposal (PIP)*. Before starting to write a PIP, make sure you follow the process [here](https://github.com/apache/pulsar/blob/master/wiki/proposals/PIP.md)
```
